packages/rum/webpack.config.js

```diff
@@ -2,9 +2,20 @@ const path = require('path')
 
 const webpackBase = require('../../webpack.base')
 
-module.exports = (_env, argv) =>
+const entry = path.resolve(__dirname, 'src/boot/rum.entry.ts')
+
+module.exports = (_env, argv) => [
   webpackBase({
     mode: argv.mode,
-    entry: path.resolve(__dirname, 'src/boot/rum.entry.ts'),
+    entry,
     filename: 'datadog-rum.js',
-  })
+  }),
+  webpackBase({
+    mode: argv.mode,
+    entry,
+    filename: 'datadog-rum-synthetics.js',
+    extraBuildEnv: {
+      SYNTHETICS_BUNDLE: 'true',
```

Comment:
Instead of a boolean, can't we pass the BUNDLE_NAME to have something like 
```
webpackBase(...{
   BUNDLE_NAME: 'DD_RUM'
}),
webpackBase(..., {
   BUNDLE_NAME: 'DD_SYNTHETICS_INJECTED_RUM'
})
```